api/src/api/taxFilingRouter.ts

```diff
@@ -1,19 +1,32 @@
+import { maskingCharacter } from "@shared/profileData";
 import { Router } from "express";
-import { TaxFilingInterface, UserDataClient } from "../domain/types";
+import { EncryptionDecryptionClient, TaxFilingInterface, UserDataClient } from "../domain/types";
 import { getSignedInUserId } from "./userRouter";
 
+const getTaxId = async (
+  taxEncryptionDecryption: EncryptionDecryptionClient,
+  taxId: string,
+  encryptedTaxId: string
+) => {
+  return encryptedTaxId === undefined && taxId.includes(maskingCharacter) === false
+    ? taxId
+    : await taxEncryptionDecryption.decryptValue(taxId);
+};
+
 export const taxFilingRouterFactory = (
   userDataClient: UserDataClient,
-  taxFilingInterface: TaxFilingInterface
+  taxFilingInterface: TaxFilingInterface,
+  taxEncryptionDecryption: EncryptionDecryptionClient
```

Comment:
our pattern is to name the variable the same as the type but lowercase. So `encryptionDecrytionClient`

This is so that the name clearly communicates what type it is. But you know it's a variable not a type because of the lowercase